src/server/components/scores.ts

```diff
@@ -0,0 +1,85 @@
+// -------------------------------------------------------------------------- //
+// Internal
+// -------------------------------------------------------------------------- //
+import { cache } from '../helpers';
+import { Scores, ScoresCollection } from '../models';
+
+// -------------------------------------------------------------------------- //
+// Functions
+// -------------------------------------------------------------------------- //
+
+export async function getAllScores() {
```

Comment:
it seems like these three functions are very similar. Can we combine them into a parametrized single function